addon/mixins/route.js

```diff
@@ -115,7 +125,15 @@ export default Ember.Mixin.create({
   */
   infinityModel(modelName, options) {
 
-    if (Ember.isEmpty(this.store) || Ember.isEmpty(this.store.find)){
+    if (emberDataVersionIs('greaterThan', '1.0.0-beta.19.2') && emberDataVersionIs('lessThan', '1.13.4')) {
```

Comment:
I agree.  I wouldn't normally do something like this^^  but the excluded versions of Ember Data were all released within a period of 3 days.  Figured anyone on one of those versions could trivially update or downgrade easily, given they're so close (and given they're genuinely bugged, that developer probably should update too).